### PR TITLE
yandex-browser: set meta.broken = false

### PR DESCRIPTION
**Pull request by dan4ik605743**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by dan4ik605743 — COMMENTED** [review]
Browser is work.
image

File: pkgs/applications/networking/browsers/yandex-browser/default.nix [hunk 1]
```diff
@@ -50,11 +50,11 @@
 
 stdenv.mkDerivation rec {
   pname = "yandex-browser";
-  version = "21.6.2.817-1";
+  version = "21.9.1.600-1";
 
   src = fetchurl {
     url = "http://repo.yandex.ru/yandex-browser/deb/pool/main/y/${pname}-beta/${pname}-beta_${version}_amd64.deb";
```
Comment (bobby285271) [inline pkgs/applications/networking/browsers/yandex-browser/default.nix:56]: Returns a 404. Shall we upload this to web.archive.org?

Comment (dan4ik605743) [reply pkgs/applications/networking/browsers/yandex-browser/default.nix:56]: > Returns a 404. Shall we upload this to web.archive.org?

The new version came out. They do not leave old links, immediately change the main one. Right now I'll fix everything.

Comment (dan4ik605743) [reply pkgs/applications/networking/browsers/yandex-browser/default.nix:56]: As for Wayback, I have long wanted to do it for a long time, but it did not work out.
image

**Review by dan4ik605743 — COMMENTED** [review]
Work
image

**Comment by dan4ik605743** [comment]
@bobby285271  check

**Comment by dan4ik605743** [comment]
im reopen

**Pull request closed without merging**